hack/pin-bundle-images.sh

```diff
@@ -46,6 +46,8 @@ for MOD_PATH in $(go list -mod=readonly -m -json all | jq -r '. | select(.Path |
 
     if [[ ${LOCAL_REGISTRY} -eq 1 && ( "$GITHUB_USER" != "openstack-k8s-operators" || "$BASE" == "$IMAGEBASE" ) ]]; then
         SHA=$(curl -s ${REPO_CURL_URL}/$BASE-operator-bundle/tags/list | jq -r .tags[] | sort -u | grep $REF)
+    elif [[ "${IMAGEREGISTRY}" != "quay.io" ]]; then
+        SHA=$(curl -s ${REPO_CURL_URL}/$BASE-operator-bundle/tag/ | jq -r .tags[].name | sort -u | grep $REF)
```

Comment:
We should land this if it unblocks CI, but something like the following could be an alternative to filter_tag_name, and avoid using sort and grep:

  jq -r '.tags | map(select ( .name | contains("$REF")))[0].name'

Reply:
I think that returns a "null" string and won't pass with [-z $SHA] check below, would leave it as is and can be worked on later, if required.

[ramishra@fedora ~]$ curl -s 'https://quay.io/api/v1/repository/openstack-k8s-operators/dataplane-operator-bundle/tag/' |  jq -r '.tags | map(select ( .name | contains("123456")))[0].name'
null

[ramishra@fedora ~]$ curl -s 'https://quay.io/api/v1/repository/openstack-k8s-operators/dataplane-operator-bundle/tag/' |  jq -r '.tags[].name' | sort -u | grep 123456
[ramishra@fedora ~]$